pkg/web_css/lib/src/_variables.scss

```diff
@@ -242,6 +242,13 @@
   .displayed-in-light-theme {
     display: none !important;
   }
+
+  a {
+    &:hover {
+      opacity: 1.0;
```

Comment:
do we have opacity other than 1?

Reply:
Yes, the version tab icons have it on 0.7 by default. However, in this case this is to counter the main rule here:
https://github.com/dart-lang/pub-dev/blob/master/pkg/web_css/lib/src/_base.scss#L98-L100

I shall move that out to a separate light-theme/dark-theme styles so we don't need the override.

Reply:
Done, PTAL.